model_analyzer/perf_analyzer/perf_analyzer.py

```diff
@@ -302,8 +300,9 @@ def _parse_perf_client_send_recv(self, section):
         if client_send_recv:
             client_send_recv = float(client_send_recv.group(1)) / 1e3
             return PerfClientSendRecv(value=client_send_recv)
-        raise TritonModelAnalyzerException(
-            'perf_analyzer output did not client send/recv time.')
+        logging.error(
+            'perf_analyzer output did not contain client send/recv time.')
+        return None
```

Comment:
Why is this return required? I think `None` is the default return value.

Reply:
It is not required. Removed.